### PR TITLE
Redesign cloudsuite web serving benchmark to use docker images

### PR DESCRIPTION
**Pull request by ivonindza**

### CODE REVIEW

**Comment by ivonindza** [comment]
ready for review

File: perfkitbenchmarker/linux_benchmarks/cloudsuite_web_serving_benchmark.py [hunk 1]
```diff
@@ -316,68 +71,108 @@ def Prepare(benchmark_spec):
...
-                 sum_ops_per_sec, 'ops/s'))
-  results.append(sample.Sample('90th percentile latency HomePage ',
-                 float(latency[0]), 'seconds'))
-  results.append(sample.Sample('99th percentile latency HomePage',
-                 float(latency99[0]), 'seconds'))
+
+  cmd = ('sudo docker run --net host --name faban_client '
+         'cloudsuite/web-serving:faban_client %s %s' %
+         (frontend.internal_ip, FLAGS.cloudsuite_web_serving_load_scale))
+  stdout, _ = client.RemoteCommand(cmd)
```
Comment (hildrum) [inline perfkitbenchmarker/linux_benchmarks/cloudsuite_web_serving_benchmark.py]: Can you put in a comment what you expect the output to look like?  It's hard to tell what you're doing here.

Comment (ivonindza) [inline perfkitbenchmarker/linux_benchmarks/cloudsuite_web_serving_benchmark.py]: The interesting part of the output is a faban summary xml. Example: http://faban.org/1.1/docs/guide/driver/samplesummary_xml.html

Comment (hildrum) [inline perfkitbenchmarker/linux_benchmarks/cloudsuite_web_serving_benchmark.py]: Could you either add a comment with a link to that URL, or paste in the text from that URL in a comment?  Not all our benchmarks do that, but some do (`iperf` for example), but it's useful.

File: perfkitbenchmarker/linux_benchmarks/cloudsuite_web_serving_benchmark.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
```
Comment (hildrum) [inline perfkitbenchmarker/linux_benchmarks/cloudsuite_web_serving_benchmark.py]: I believe this should be 2015-2016

**Comment by ivonindza** [comment]
updated

**Comment by hildrum** [comment]
LGTM.  Let me know when you want me to pull it in.

**Comment by ivonindza** [comment]
Go ahead, pull it in. Thanks.

**Pull request merged**